index.js

```diff
@@ -35,13 +42,13 @@ Ewma.prototype.insert = function insert(x) {
     var elapsed = now - self._stamp;
     self._stamp = now;
 
-    var w = Math.exp(-elapsed / self._tau);
+    var w = Math.pow(2, -elapsed / self._decay);
```

Comment:
I believe that `pow(a, b)` is implemented in term of `exp(b * log(a))` so that's basically just doing the same thing.
It's slightly less efficient, but if you think it makes the code clearer you can keep it that way.

Ref: https://stackoverflow.com/questions/17891595/pow-vs-exp-performance